ontology/spdx-ontology.owl.xml

```diff
@@ -706,40 +673,6 @@ Known issues:</rdfs:comment>
     
 
 
-    <!-- http://www.w3.org/2000/01/rdf-schema#member -->
-
-    <owl:ObjectProperty rdf:about="http://www.w3.org/2000/01/rdf-schema#member"/>
-    
-
-
-    <!-- http://www.w3.org/2009/pointers#endPointer -->
-
-    <owl:ObjectProperty rdf:about="http://www.w3.org/2009/pointers#endPointer">
-        <rdfs:domain rdf:resource="http://www.w3.org/2009/pointers#CompoundPointer"/>
-        <rdfs:range rdf:resource="http://www.w3.org/2009/pointers#SinglePointer"/>
-        <ns:term_status xml:lang="en">stable</ns:term_status>
-    </owl:ObjectProperty>
-    
-
-
-    <!-- http://www.w3.org/2009/pointers#reference -->
-
-    <owl:ObjectProperty rdf:about="http://www.w3.org/2009/pointers#reference">
-        <rdfs:domain rdf:resource="http://www.w3.org/2009/pointers#SinglePointer"/>
-        <ns:term_status xml:lang="en">stable</ns:term_status>
-    </owl:ObjectProperty>
-    
-
-
-    <!-- http://www.w3.org/2009/pointers#startPointer -->
-
-    <owl:ObjectProperty rdf:about="http://www.w3.org/2009/pointers#startPointer">
-        <rdfs:domain rdf:resource="http://www.w3.org/2009/pointers#CompoundPointer"/>
-        <rdfs:range rdf:resource="http://www.w3.org/2009/pointers#SinglePointer"/>
-        <ns:term_status xml:lang="en">stable</ns:term_status>
-    </owl:ObjectProperty>
-    
-
```

Comment:
Agree this should be removed from the OWL document to prevent redefinitions.  This will, however, cause a problem in some of the SPDX tools implementations that depends on having a definition of these types/classes in the OWL document.  I'll need to look into how to include the OWL definitions in a separate file, or ideally reference a pre-existing definition from this OWL document.